File: my-messenger-backend/my.messenger.common/Users/Group.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace my.messenger.common.Users
{
    public class Group
    {
        public String Id { get; set; }
        public String Name { get; set; }
        public String OwnerUserId { get; set; }
        public List<String> Members { get; set; }

        public Group()
        {
            Id = Identifier.New();
        }
    }
}

File: my-messenger-android/app/src/main/java/my/messenger/androidclient/api/model/Group.java
package my.messenger.androidclient.api.model;

public class Group {
    private String id;
    private String name;
    private String ownerUserId;
    private String[] members;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getOwnerUserId() {
        return ownerUserId;
    }

    public void setOwnerUserId(String ownerUserId) {
        this.ownerUserId = ownerUserId;
    }

    public String[] getMembers() {
        return members;
    }

    public void setMembers(String[] members) {
        this.members = members;
    }
}
